scripts/generate-entry-i10n-test.js

```diff
@@ -0,0 +1,25 @@
+// Get google translate credentials
```

Comment:
@paninee do we need this file? if not let's delete it.

Reply:
Done